docs/markdown/Using Pants/concepts/options.md

```diff
@@ -206,6 +206,18 @@ listopt = [
 ]
 ```
 
+You can append elements to any existing elements that may have been defined in lower-precedence sources. In this example, having the config file above, there will be a list of three elements at the runtime:
+
+```bash
+./pants --scope-listopt=baz
+```
+
+It is also possible to override any existing list values provided. In this example, having the config file above, there will be a list of one element at the runtime:
+
+```bash
+./pants --scope-listopt="['baz']"
+```
```

Comment:
Maybe instead of putting this in the "Config file section", move it up a few lines in the "Command line flags" section? We already explain the difference between `[]` vs implicit add, only don't do a good job at it. Your explanation is better.

Reply:
I like it better in the "Config file entries" section because the example is specifically for the overriding an existing config file. If I move it elsewhere, I have to move the pants.toml contents as well :/

Reply:
How about on line 198, you say:

> Appending will add to any values from lower-precedence sources, such as config files (`pants.toml`) and possibly Pants's `default`. Otherwise, using `[]` will _override_ any lower-precedence sources.

--

> because the example is specifically for the overriding an existing config file. If I move it elsewhere, I have to move the pants.toml contents as well 

I'm not sure the example would be necessary?